cinema/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+# Create your models here.
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.CharField(max_length=255)
```

Comment:
The 'description' field is defined as a CharField with a max_length of 255, which is appropriate according to the checklist: 'If you specify max_length then it's more reasonable to use CharField instead of TextField.'